src/plugins/anki/anki-plugin.tsx

```diff
@@ -0,0 +1,291 @@
+import { Button } from "@components/Button";
+import {
+  AnoriPlugin,
+  WidgetConfigurationScreenProps,
+  OnCommandInputCallback,
+  WidgetRenderProps,
+  WidgetDescriptor,
+} from "@utils/user-data/types";
+import "./styles.scss";
+import { getAllWidgetsByPlugin } from "@utils/plugin";
+import { translate } from "@translations/index";
+import { useTranslation } from "react-i18next";
+import { useEffect, useState } from "react";
+import { Select } from "@components/Select";
+
+type AnkiPluginWidgetConfigType = {
+  deckName: string;
+  deckId: number;
+};
+
+function invoke(action: string, version: number, params = {}) {
+  return new Promise((resolve, reject) => {
+    const xhr = new XMLHttpRequest();
```

Comment:
Is there a reason why you're using XMLHttpRequest instead of fetch? If AnkiConnect works with fetch, please change this to fetch

Reply:
was given in the ankiconnect documentation, but I will change it